### ISSUE TITLE
Albumin low in ascitic fluid relative to serum

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10563]
**Preferred term label:**
Albumin low in ascitic fluid relative to serum

**Synonyms**
Serum-Ascites Albumin Gradient

**Definition (free text, please give PubMed ID)**
difference between albumin in the serum and albumin in the ascitic fluid with an elevated level defined as greater than 1.1 g/dL

**Parent term (use hpo.jax.org/app)**
https://uts.nlm.nih.gov/uts/umls/concept/C4331276

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bacterial peritonitis
Cirrhosis of various types
Cardiac amyloidosis
Cardiac valvular abnormalities of various types

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000863
- label: Elevated serum-ascites albumin gradient (SAAG)
- parents: [Abnormality of digestive system physiology (HP:0025032)](https://hpo.jax.org/app/browse/term/HP:0025032)
- pmid: PMID:29201716
- def: An elevation of the serum ascites albumin gradient (SAAG), which is based on the difference between the albumin level of serum and of ascitic fluid, may be used to assess the extent of ascites. The SAAG is thought to reflect the colloid osmotic pressure gradient and the degree of portal hypertension. It has been shown that SAAG is a better discriminator of portal hypertension than ascites. A SAAG of 1.1 gm/dL or higher is considered as high, indicating the presence of portal hypertension, while SAAG less than 1.1 gm/dL is low, indicating the absence of portal hypertension.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10563
- synonyms: Abnormally high serum-ascites albumin gradient (SAAG)(NONE)
